### ISSUE TITLE
NullPionterException Crash When Exiting While Picker is Loading

### ISSUE THREAD

**Issue by alaashaheen** [WindSekirun/NaraeImagePicker#5]
The following error occurs when you exit (on back press) while photo selector is loading:

```
FATAL EXCEPTION: main Process: work.alaa.pinnedphoto,  PID: 12049
java.lang.NullPointerException: Attempt to invoke virtual method 'void pyxis.uzuki.live.naraeimagepicker.widget.EmptyRecyclerView.notifyDataSetChanged()' on a null object reference
    at pyxis.uzuki.live.naraeimagepicker.fragment.AlbumFragment$loadItem$1.invoke(AlbumFragment.kt:70)
    at pyxis.uzuki.live.naraeimagepicker.fragment.AlbumFragment$loadItem$1.invoke(AlbumFragment.kt:24)
    at pyxis.uzuki.live.richutilskt.utils.RichUtils__RThreadKt$sam$Runnable$f44beeba.run(Unknown Source:2)
    at android.os.Handler.handleCallback(Handler.java:789)
    at android.os.Handler.dispatchMessage(Handler.java:98)
    at android.os.Looper.loop(Looper.java:164)
    at android.app.ActivityThread.main(ActivityThread.java:6809)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```
![picker_screen_360](https://user-images.githubusercontent.com/9418408/36069324-51c4b1f8-0eb5-11e8-845c-603f6b3d04cb.png)

**Comment by WindSekirun** [comment]
Ok, i'll check

**Comment by WindSekirun** [comment]
fixed in 1.1.1 . thanks!
